src/main/java/com/hazelcast/kubernetes/RetryUtils.java

```diff
@@ -58,8 +58,8 @@ private RetryUtils() {
                 }
                 long waitIntervalMs = backoffIntervalForRetry(retryCount);
                 LOGGER.warning(
-                        String.format("Couldn't connect to the Kubernetes master, [%s] retrying in %s seconds...", retryCount,
-                                waitIntervalMs / MS_IN_SECOND));
+                        String.format("Couldn't find Hazelcast members using Kubernetes API, [%s] retrying in %s seconds...",
```

Comment:
could be `discover` a better word but not sure.. Apart from that, why are we trying to query 10 times if we are able to get an answer from k8s api?. It is because we want to discover at least one other member to form the cluster?

Reply:
I also see that in the related issue, `service-port` usage is causing some confusion... There would not be a problem if it was not used at all..

Reply:
Yeah, we should probably get rid of `service-port` at some point. We are retrying in case Kubernetes master is busy or something. But 10 times is too much I guess. There is already a GH Issue to reduce it: #130.

Changed `find` => `discover`.